css-pseudo-classes - Matthew Kweon

MattKweon opened the pull request:
![Screen Shot 2022-08-24 at 10 00 23 AM](https://user-images.githubusercontent.com/105886708/186479094-244da62f-3577-426b-8d57-4391b833635f.png)

thebearingedge reviewed (changes requested):
The user's cursor should change to a pointing hand when they hover over the button.

MattKweon commented:
Added the pointing hand when hovering over the button. Also changed css code for the striped table to use a pseudo-class selector instead of assigned id's and class to the <tr> element.
